streams/piping/pipe-through.js

```diff
@@ -30,6 +30,8 @@ function duckTypedPassThroughTransform() {
   };
 }
 
+const uninterestingReadableWritablePair = { writable: new WritableStream(), readable: new ReadableStream() };
```

Comment:
though currently this is fine, i suggest that this kind of shared test data to be created for each test cases in principle. how about making this a factory function?